swift/apple/FirezoneNetworkExtension/Adapter.swift

```diff
@@ -520,13 +514,3 @@ extension Adapter: CallbackHandlerDelegate {
     }
   }
 #endif
-
-extension Network.NWPath {
-  func connectivityDifferentFrom(path: Network.NWPath) -> Bool {
```

Comment:
The alternative here is to add `supportsDNS` to the list of connectivity changes we care about. There are lots of other path properties I'm not sure we need to respond to. This is safer w.r.t. resets, but runs the risk of resetting too often.

Reply:
Ok I did the above here. Looks like many new properties are being added to the path struct here in macOS 26 that we won't want to act upon, like link quality changes.